va_explorer/va_data_management/management/commands/run_dhis.py

```diff
@@ -0,0 +1,242 @@
+from django.core.management.base import BaseCommand
+from va_explorer.va_data_management.models import VerbalAutopsy, CauseOfDeath, dhisStatus,cod_codes_dhis
+from django.forms.models import model_to_dict
+from io import StringIO
+from collections import OrderedDict
+import dateutil.parser,os,requests,collections
+import csv, environ
+# from . import dhis as dhis
```

Comment:
import seems to be a remnant from local testing. Consider removing.

Reply:
done